client/components/data/purchases/edit-card-details/index.jsx

```diff
@@ -32,7 +32,7 @@ function getStateFromStores( props ) {
 
 function isDataLoading( state ) {
 	return (
-		! state.card ||
+		state.card.isFetching ||
```

Comment:
As far as I can tell, the issue about error handling is not the root cause of the bug that #1264 was opened to track. I opened an issue to track improvements to error handling so that we can continue that discussion: https://github.com/Automattic/wp-calypso/issues/1776